soft404.py

```diff
@@ -7,9 +7,7 @@
 
 """
 
-import difflib
-import httplib
-import random
+import difflib, httplib, random
```

Comment:
Why the reformatting change?

Reply:
Good question. I thought long and hard about this one, and came to the conclusion that it was necessary.